test/sbd.metrics

```diff
@@ -0,0 +1,4 @@
+# HELP ha_cluster_sbd_device_status Health status for each SBD device; 1 is healthy, 0 is not
+# TYPE ha_cluster_sbd_device_status gauge
+ha_cluster_sbd_device_status{device_name="/dev/vdc"} 1 1
```

Comment:
why is there `1 1` here?

Reply:
The latter 1 is the timestamp in millisecond. All tests use the same timestamp, initialized here: https://github.com/ClusterLabs/ha_cluster_exporter/blob/81da50bf06cd9e5e3502f0c201226099151c6122/ha_cluster_exporter_test.go#L15-L19

Reply:
I will make this more clear with a different value, I now realise that `1` is not very expressive.